README.new-mail

```diff
@@ -0,0 +1,48 @@
+new-mail-command Patch
+======================
+
+    Configure a custom command to be executed when a new email is received.
+
+Patch
+-----
+
+    To check if Mutt supports "new-mail-command", look for
+    "new-mail" in the mutt version.
```

Comment:
Mutt seems to be supporting it, even if I do not get `new-mail` (or similar) in the output of `mutt -v`. Nor could I find any option to enable/disable it at compile time. Shouldn’t the only thing to look for be Mutt >= 1.6.2, as in the dependencies list?